### ISSUE TITLE
LicenseWebpackPlugin is incompatible with meteor-client-bundler

### ISSUE THREAD

**Issue by darkbasic** [xz64/license-webpack-plugin#18]
`angular-cli` uses `LicenseWebpackPlugin` when building for production:
```
    new LicenseWebpackPlugin({
      "licenseFilenames": [
        "LICENSE",
        "LICENSE.md",
        "LICENSE.txt",
        "license",
        "license.md",
        "license.txt"
      ],
      "perChunkOutput": false,
      "outputTemplate": "/home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/output.template.ejs",
      "outputFilename": "3rdpartylicenses.txt",
      "suppressErrors": true,
      "includePackagesWithoutLicense": false,
      "abortOnUnacceptableLicense": false,
      "addBanner": false,
      "bannerTemplate": "/*! 3rd party license information is available at <%- filename %> */",
      "includedChunks": [],
      "excludedChunks": [],
      "additionalPackages": [],
      "pattern": /^(MIT|ISC|BSD.*)$/
    }),
```

Unfortunately this causes the following error when using [meteor-client-bundler](https://github.com/Urigo/meteor-client-bundler/):
```
$ npm run start

> angularcli-meteor@0.0.0 start /home/niko/WebstormProjects/angularcli-meteor
> webpack-dev-server --port=4200

 10% building modules 3/3 modules 0 activeProject is running at http://localhost:4200/
webpack output is served from /
404s will fallback to /index.html
 95% emittingUnhandled rejection Error: ENOTDIR: not a directory, open '/home/niko/WebstormProjects/angularcli-meteor/node_modules/meteor-client.js/package.json'
    at Object.fs.openSync (fs.js:652:18)
    at Object.fs.readFileSync (fs.js:553:33)
    at LicenseExtractor.readPackageJson (/home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/dist/LicenseExtractor.js:121:23)
    at LicenseExtractor.parsePackage (/home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/dist/LicenseExtractor.js:21:32)
    at ModuleProcessor.processPackage (/home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/dist/ModuleProcessor.js:24:46)
    at ModuleProcessor.processFile (/home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/dist/ModuleProcessor.js:21:21)
    at moduleCallback (/home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/dist/LicenseWebpackPlugin.js:80:61)
    at Set.forEach (native)
    at Chunk.forEachModule (/home/niko/WebstormProjects/angularcli-meteor/node_modules/webpack/lib/Chunk.js:159:17)
    at /home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/dist/LicenseWebpackPlugin.js:88:27
    at Array.forEach (<anonymous>)
    at Compiler.<anonymous> (/home/niko/WebstormProjects/angularcli-meteor/node_modules/license-webpack-plugin/dist/LicenseWebpackPlugin.js:65:32)
    at next (/home/niko/WebstormProjects/angularcli-meteor/node_modules/tapable/lib/Tapable.js:204:14)
    at /home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/index.js:204:9
    at PassThroughHandlerContext.finallyHandler (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/finally.js:56:23)
    at PassThroughHandlerContext.tryCatcher (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/home/niko/WebstormProjects/angularcli-meteor/node_modules/html-webpack-plugin/node_modules/bluebird/js/release/async.js:143:10)
```

Commenting out `import 'meteor-client';` from `src/main.ts` solves the building issue, but obviously the app will not work.

You can easily reproduce the issue checking out my [angularcli-meteor tutorial repository](https://github.com/darkbasic/angularcli-meteor).

**Comment by xz64** [comment]
I am getting this error when I try to build the repository you linked to:

```
ERROR in ./src/main.ts
Module not found: Error: Can't resolve 'meteor-client' in '/tmp/angularcli-meteor/src'
 @ ./src/main.ts 1:0-23
```

It looks like meteor-client is not found in node_modules. Can you explain how the import to meteor-client gets resolved?

**Comment by darkbasic** [comment]
I'm sorry, I forgot to explain that part.
First you will have to install Meteor:
```
curl https://install.meteor.com/ | sh
```
Then `meteor-client-bundler`:
```
npm install -g meteor-client-bundler
```
Finally, inside the project folder, you will have to generate the meteor-client bundle:
```
meteor-client bundle -s api
```
Now you should be able to run the project with `npm run start` or `npm run build`.

**Comment by xz64** [comment]
@darkbasic  I made a change to fix this issue. Try out the dev branch and let me know if it working fine you. You can run `npm install`, `npm run build` followed by `npm pack` to generate a .tgz file which you can then install in your app via `npm install  ../path/to/license-webpack-plugin-1.0.0.tgz`

**Comment by darkbasic** [comment]
Works flawlessly, thanks. Can you please tag a new version and release it on npm?

**Comment by xz64** [comment]
Published in 1.0.1

**Comment by darkbasic** [comment]
Thank you so much.
